guestfs-ocaml.m4: Check caml_alloc_initialized_string

Always check, even if --disable-ocaml is selected

---
 m4/guestfs-ocaml.m4 | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/m4/guestfs-ocaml.m4 b/m4/guestfs-ocaml.m4
index 18f95c2..bf07799 100644
--- a/m4/guestfs-ocaml.m4
+++ b/m4/guestfs-ocaml.m4
@@ -214,8 +214,7 @@ AM_CONDITIONAL([HAVE_BYTES_COMPAT_ML],
 	       [test "x$have_Bytes_module" = "xno"])
 
 dnl Check if OCaml has caml_alloc_initialized_string (added 2017).
-AS_IF([test "x$OCAMLC" != "xno" && test "x$OCAMLFIND" != "xno" && \
-       test "x$enable_ocaml" = "xyes"],[
+AS_IF([test "x$OCAMLC" != "xno" && test "x$OCAMLFIND" != "xno"],[
     AC_MSG_CHECKING([for caml_alloc_initialized_string])
     cat >conftest.c <<'EOF'
 #include <caml/alloc.h>
-- 
2.31.1

